workflows/reference_files/2.4/reference_files_header.smk

```diff
@@ -76,11 +76,17 @@ for build_name, build_info in config["genome_builds"].items():
         f"`provider` not set for `{build_name}` or `provider` not among {possible_providers}."
     )
     if "genome_fasta_url" in build_info:
```

Comment:
this part has been a problem needing an update already for long! Since you modified it here, can you please instead drop the whole url_code ping?

Reply:
I've deleted all the HTTP 200 checks